src/osm_configurator/view/toplevelframes/calculation_frame.py

```diff
@@ -404,19 +406,19 @@ def __end_calculation_successfully(self):
                                     text_color=button_constants_i.ButtonConstants.BUTTON_TEXT_COLOR.value
                                     )
         self.visualize_button.grid(column=2, row=2, rowspan=1, columnspan=1, padx=10, pady=10)
-        self.resetable_elements.append(self.visualize_button)
+        self.resettable_elements.append(self.visualize_button)
         self.buttons.append(self.visualize_button)
 
         self.progressbar.set(1)
         self.__color_buttons_with_int(5)
         self.__activate_buttons()
         self.buttons.remove(self.cancel_button)
-        self.resetable_elements.remove(self.cancel_button)
+        self.resettable_elements.remove(self.cancel_button)
         self.cancel_button.destroy()
         self._state_manager.unlock_state()
 
-
-    def __get_next_phase(self, current_phase: CalculationPhase) -> CalculationPhase:
+    @staticmethod
+    def __get_next_phase(current_phase: CalculationPhase) -> CalculationPhase:
```

Comment:
idk know if making some random static methods in the middle is actually good or not